default.nix

```diff
@@ -19,7 +19,6 @@ let
   cardano-sl-src = builtins.fromJSON (builtins.readFile ./cardano-sl-src.json);
   cardano-sl-pkgs = import (pkgs.fetchgit cardano-sl-src) {
                       gitrev = cardano-sl-src.rev;
-                      inherit dconfig;
```

Comment:
You should have merged https://github.com/input-output-hk/iohk-ops/pull/146 to fix this - I'll do it now